plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -112,6 +113,10 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_TRUNCATE:
                 return true;
 
+            case SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN:
+                // TODO support pushdown of complex expressions in predicates
```

Comment:
not filing issue for this one. it's impossible to miss the fact, JDBC connectors do not support complex expression predicate pushdown